tardis/io/schemas/montecarlo.yml

```diff
@@ -69,8 +69,8 @@ properties:
     default: {}
     properties:
       track_rpacket:
-        type: boolean
```

Comment:
Please remove all changes from the schema files.  Tracking the r-packets is something that is set by the user in their own config file.